view_distillation/design_doc.md

```diff
@@ -0,0 +1,69 @@
+# View Distillation Design Doc
+
+## Input
+
+- A set of views
+  - For now, only allow views to be in csv format
+  - Can expand to different formats (ex: from databases) in the future
+
+## Create an instance
+
+`vd = ViewDistillation(path_to_views)`
+
+## APIs
+
+(The views are returned as paths.)
+
+### Find compatible views
+
+`result = vd.find_compatible_views()`
+
+This will return a list of lists, each inner list contains the views that are identical to each other.
+
+### Prune compatible views
+
+`result = vd.prune_compatible_views()`
```

Comment:
a better name would suggest what the summarization operation does, e.g., reduce_compatible_views_to_one()

the pruning aspect is implicit in the operation, but this name explains how the pruning is done